### PR TITLE
Docs plus cleanup

### PR DESCRIPTION
**Pull request by kisst**

### CODE REVIEW

File: javascripts/mindmap.js [hunk 1]
```diff
@@ -381,15 +381,15 @@ function update(source) {
...
+    });
 
   nodeEnter.append("circle")
     .attr('class', 'nodeCircle')
     .attr("r", 0)
     .style("fill", function (d) {
       return d._children ? "lightsteelblue" : "#fff";
-    });
+    })
+    .on('click', click);
```
Comment (evenicoulddoit) [inline javascripts/mindmap.js:392]: What's this?

File: convert_from/xmind.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        else:
+            nodeattrs[XML_NODE_CONTENT] = node.text
+    depth += 1
+
+    for child in children:
+        yamlout(child, depth)
+
+
+if __name__ == "__main__":
+    main()
```
Comment (evenicoulddoit) [inline convert_from/xmind.py:59]: kewl

**Review by evenicoulddoit — COMMENTED** [review]
Just one question :)

File: javascripts/mindmap.js (hunk 1 above)
Comment (kisst) [reply javascripts/mindmap.js:392]: So before, the click was on the circle, and the text, aka the node, now it's only the circle, ideally I would love to slip the set focus, update breadcrumbs, and the open/close click actions, but that's beyond me, so I just did this patch to start with.

File: convert_from/xmind.py (hunk 2 above)
Comment (kisst) [reply convert_from/xmind.py:59]: Well well, that whole is just .... draft, not even functional, the whole is just generating a print, not processing any formatting, only the first sheet, etc etc, Just wanted to add it as a first (almost) working version

**Pull request merged**